hugo_site/content/visa.md

```diff
@@ -1,5 +1,5 @@
 # Visa
-If your country is not part of the [Schengen Agreement](https://www.schengenvisainfo.com/who-needs-schengen-visa/), you will need to request a tourist Visa in the Danish Embassy. We have spoken to the Danish Immigration Office and below we have compiled a list of attachments you will need to provide when requesting the travel document. Have in mind that this process can take up to 90 days, therefore we recommend starting it as soon as possible.
+If your country is not part of the [Schengen Agreement](https://www.schengenvisainfo.com/who-needs-schengen-visa/), you will need to request a tourist Visa in the Danish Embassy. We have spoken to the Danish Immigration Office and below we have compiled a list of attachments you will need to provide when requesting the travel document. Contact the local embassy for information on processing time.
```

Comment:
Was this part incorrect since you removed it?

> Have in mind that this process can take up to 90 days

Reply:
I found out the processing time depends on each embassy. They are the ones processing these visas. Only upper level requests are dealt by Udlændingestyrelsen, such as work and residence permits. So lets leave it unmentioned.

Reply:
That's cool - yeah probably people know their own countries issues and timelines better than we can tell or hint.